### ISSUE TITLE
Neukonzeption der grafischen Benutzeroberfläche

### ISSUE THREAD

**Issue by windauer** [humboldtdigital/ProHD#20]

**Comment by windauer** [comment]
@arojascastro will provide input

**Comment by tuurma** [comment]
@arojascastro the sooner we can get this information, the better; in particular we'd need to:

- any logos or other images for the page header/footer, backgrounds to use etc
- color palette: RGB codes with information where to apply them (backgrounds, borders, font colors etc.)
- listing of fonts: font-family and sizes for different scenarios (transcription text, headings of various levels, highlights, pop-up commentary text, menu items ...)
- sketch or mockup of any layout changes

**Comment by arojascastro** [comment]
A redesign of the graphical user interface (landing page) similar to "Vincent van Gogh - The Letters (https://teipublisher.com/exist/apps/vangogh/index.html). This new interface should be adapted to the project's color scheme (https://nubes.bbaw.de/s/qWpMnn3iK628Y4o), include the project logo, and provide access to the editions and other pages (about the project, the team, etc.). 

## Colors

- Primary green: 0, 143, 143
- Black olive: 53, 56, 49
- Grey: 242, 240, 235
- Secondary turquoise: 122, 199, 196
- Secondary azure: 226, 244, 241
- Orange yellow: 244, 189, 51
- Burnt Sienna: 238, 108, 77

Please bear in mind that the alternative secondary used in a background is sometimes lacking enough contrast if used with white.

## Fonts

- Montserrat Semi Bold
- Montserrat Light

I think the current font size is good enough.

I will share with you the manual via email. 

## Mockup

I will try to design an mockup next week and send it to you as well via email. (Perhaps you can work on the other tickets in the meantime.)

**Comment by tuurma** [comment]
We'll start with adding a landing page following the mockup at https://app.moqups.com/s9KP0THBCQUBvVAm9kOnrUOLOkpkfhhx/view/page/a4eb26dd7

other changes we'll implement on the best effort basis, depending on the time constraints

Desiderata re visual aspects from Antonio:

> Guidelines for the Redesign of the GUI of Digital Dossier: Alexander von Humboldt & Cuba (1800-1830)
By following these guidelines, we aim to create a consistent and visually appealing graphic user interface for Proyecto Humboldt Digital's Digital Dossier. It is essential to maintain the colour and font choices throughout the website to ensure that it is easy to navigate and understand for all users. Additionally, the use of rounded corners can create a friendly and approachable design that fits with the project's values. The current graphic user interface of the Digital Dossier is responsive for mobiles and tablets and meets the main accessibility principles. We expect that the new resulting GUI's will also be responsive and accessible, as we place great importance on providing an inclusive experience for all users, regardless of their device or ability. Our goal is to create a design that not only looks visually appealing but also meets the needs of all users, including those with disabilities. 
Mock-ups 
Link: https://app.moqups.com/s9KP0THBCQUBvVAm9kOnrUOLOkpkfhhx/view/page/ad64222d5
Please make sure that you zoom out to see our current 10 mock-ups and be aware of additional comments.
Fonts
Use Roboto Slab Semi Bold for the main title (28) and the title of the editions (18)
Use Roboto Slab regular for the text of the editions (14).
Use Montserrat Bold for the titles (18) of sections in the main menu.
Use Montserrat Semi Bold for labels (16) and help texts (14) in bars.
Use Montserrat Regular (14) for main texts in other pages (Landing page, about, etc.).
Colours
Use black olive (#353831) for the title of the website, title of each edition, labels, summary, and texts.
Use green (#008f8f) for the labels of each section of the website in the main menu.
Use azure green (#e2f4f1) as the background colour.
Use medium turquoise (#7ac7c4) for the main box background.
Use grey (#f2f0eb) in the main bar of the footer and again green (#008f8f) in the second bar of the footer.
Other Remarks
Add rounded corners to boxes and bars.
Add simple icons next to labels when possible (i.e. Search bar).
Use 1.4 Line Height for the text of editions. 1.3 can be used in the licence notice in the footer.
For mobile design: use the same fonts, bur reducing the font size: until 18 for the title of the dossier. The minimum size should be 14 in main body of the text. Use icons whenever possible. In the edition page, place the facsimile after the text at the bottom (as it is in our current design.)
Do not forget alternative-texts when using images as in our current interface.
Use the favicon of the project and the logo when loading new pages as in our current GUI’s.
Avoid using white and grey when using azure green (#e2f4f1) in the background for contrast reasons. White can only be used if the background colour is #008f8f also for contrast reasons.

**Comment by arojascastro** [comment]
Hi, thanks for your work!

We are reviewin the GUI redesign and have some comments:

- On desktop: In the browsing area, there is a problem when clicking on the XML format icon. We can only access to the XML from inside the edition, but not from the browsing area (sometimes it works, sometimes it does not).
- On IPad, none of the XML icons are giving access to the XML page.
- On Desktop: in static pages and editions, there is a redundancy: there is both a burger menu and a menu bar. The burger menu is not necessary on Desktop.
- General note: we suggest replacing the < > icon with a term: "XML". The < > icon is a bit confusing, it is too similar to next and previous page.

Looking forwards to hearing from you! Thanks

**Comment by tuurma** [comment]
Thanks!

**Comment by tuurma** [comment]
Dear Antonio, we couldn't reproduce the ipad problem, could you check which browser you're using and doublecheck that the isssue persists?

We adressed other remarks and hope things are looking good enough to show it in Graz. NB the Seitennavigation now works, you can search for a page label (e.g. 5r) or use wildcards for partial matches.

**Comment by arojascastro** [comment]
Thanks! @trafikante can give us more details about the browsing he used to test it with his IPad.

Thanks for the navigation droplist. Can you create a pull request and merge the changes so we can update the web application in our servers? 

Thanks!

**Comment by trafikante** [comment]
I used an iPad Pro (1st generation) with Firefox and Safari. Both had problems with the XML icon (it wouldn't load the xml or do anything at all). I also had problems with the citation popup that only 1 out of 5 times (roughly) would stay after I touched it and otherwise disappear inmediately (which means, that I couldn't really read what it said). Unfortunately, I can't reproduce this now (during Graz), since it wasn't my iPad and I have only acces to it after Graz.

But thank you, @tuurma for working so hard on making the Edition better before we showcase it!
